Add missing search extension to product listing loader

acris-cp opened the pull request:
By Shopware 6.2.0-RC1 the search extension is no longer available for products in the product listing.
Inside the search extension the _score is stored which is needed for the Shopware Enterprise Search.


### 1. Why is this change necessary?
The search extension and the _score is missing for the product search result. Shopware Enterprise Search needs the _score for the search preview.
For the change of Shopware 6.2.0-RC1 this missing.

### 2. What does this change do, exactly?
It adds the search extension with the _score to the search result.

### 3. Describe each step to reproduce the issue or behaviour.
Try the search preview of the Shopware Enterprise Search at 6.2.0-RC1.

### 4. Please link to the relevant issues (if any).
No issue was created yet.

### 5. Checklist

- [ ] I have written tests and verified that they fail without my change
- [x] I have squashed any insignificant commits
- [ ] I have written or adjusted the documentation according to my changes
- [ ] This change has comments for package types, values, functions, and non-obvious lines of code
- [x] I have read the contribution requirements and fulfil them.

hlohaus commented:
Hi,

thanks for your PullRequest.

Can you fix the "StyleCI" check?

 https://styleci.io/analyses/q5APnP

Greeting Heiner

acris-cp committed:
Add missing search extension to product listing loader

By Shopware 6.2.0-RC1 the search extension is no longer available for products in the product listing.
Inside the search extension the _score is stored which is needed for the Shopware Enterprise Search.

ssltg commented:
@acris-cp thank you for your condribution. 👍 
The search extension was readded already into 6.2: https://github.com/shopware/platform/blob/6.2/src/Core/Content/Product/SalesChannel/Listing/ProductListingLoader.php

So we will close this PR.

Regards
